### ISSUE TITLE
[Windows] Constant high CPU usage

### ISSUE THREAD

**Issue by romenoo** [monero-project/monero-core#757]
The monero-wallet-gui.exe process is constanly using high CPU without any reason, even when not connected to the network. Could somebody please help? Is this a bug? I'm not mining or anything

**Comment by Jaqueeee** [comment]
Sounds like a bug yes. Are you connected to a local or remote node? What's your computer and os specs?

**Comment by romenoo** [comment]
No remote node, and no connection necessary for this bug, the moment the GUI shows up, high CPU usage starts with no end. Windows 7 x64, plenty of RAM and CPU

**Comment by romenoo** [comment]
I found a way to stop this, after I start monero-wallet-gui.exe If I block all connections to monerod.exe, when it says "Daemon failed to start" after the failed connection attempts the CPU usage stops. Otherwise when there are connection attempts or a successful connection to monerod.exe, there is constant CPU usage.

Using Process Hacker I found that there is also constant GPU usage along with CPU usage and that stops too after blocking monerod.exe connections.

In case it's related, these lines show up in monero-wallet-gui.log when I start monero-wallet-gui.exe

WARN 	net.dns	src/common/dns_utils.cpp:531	WARNING: no two valid MoneroPulse DNS checkpoint records were received
ERROR	WalletAPI	src/wallet/api/wallet.cpp:551	Status_Critical - not storing wallet
